pkg/api/v2/reporter/reporter_test.go

```diff
@@ -36,6 +37,7 @@ var _ = Describe("Reporter", func() {
 		mockResourceClient = memory.NewResourceClient(memory.NewInMemoryResourceCache(), &v1.MockResource{})
 		fakeResourceClient = memory.NewResourceClient(memory.NewInMemoryResourceCache(), &v1.FakeResource{})
 		reporter = rep.NewReporter("test", statusClient, mockResourceClient, fakeResourceClient)
+		rep.DisableTruncateStatus = false
```

Comment:
might be helpful to place a comment as to why `TruncateStatus = false`

Reply:
I like it, updated